UploadDaemon/service/UploadDaemonService.xml

```diff
@@ -0,0 +1,33 @@
+<service>
+	<id>teamscale-dotnet-profiler-upload-daemon</id>
+	<name>Teamscale DotNet Profiler UploadDaemon</name>
+	<description>Teamscale DotNet Profiler UploadDaemon</description>
+
+	<!-- Adjust these paths to your setup. -->
+	<!-- PROFILER_DIR should be the path that contains Profiler32.dll -->
+	<env name="PROFILER_DIR" value="c:\path\to\profiler" />
+
+	<executable>%PROFILER_DIR%\UploadDaemon\UploadDaemon.exe
```

Comment:
Is this trailing newline ok?

Reply:
Yes, was like that for Teamscale as well